### PR TITLE
Client footer refactor

### PR DESCRIPTION
**Pull request by YarkoSumyk**

### CODE REVIEW

**Review by bandvov — APPROVED** [review]

File: client/src/configs/frontend-config.js [hunk 1]
```diff
@@ -47,10 +47,11 @@ export const deliveryType = {
 };
 export const _apiBase = 'http://localhost:5000/';
 
-export const socialNetworksURL = {
+export const socialNetworkLinks = {
+    title: 'Links',
     telegram: 'https://web.telegram.org',
     facebook: 'https://www.facebook.com/Fn-100171608356044/',
-    instagram: 'https://www.instagram.com/'
+    instagram: "https://www.instagram.com/"
```
Comment (VolodymyrTrach) [inline client/src/configs/frontend-config.js]: Change '' to '

**Review by VolodymyrTrach — CHANGES_REQUESTED** [review]

**Comment by VolodymyrTrach** [comment]
Pull request should be to development branch not to master

File: client/src/configs/frontend-config.js [hunk 2]
```diff
@@ -47,10 +47,11 @@ export const deliveryType = {
 };
 export const _apiBase = 'http://localhost:5000/';
```
Comment (oleksandrhodzevych) [inline client/src/configs/frontend-config.js:48]: remove hardcoded 5000 то env

**Review by oleksandrhodzevych — CHANGES_REQUESTED** [review]

File: client/src/components/app-footer-links/App-footer-links.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            <div>
+                <a className='footer-icon' href={socialNetworkLinks.telegram} target='_blank'>
+                    <FontAwesomeIcon className="icon" icon={faTelegramPlane} />
+                </a>
+                <a className='footer-icon' href={socialNetworkLinks.instagram} target='_blank'>
+                    <FontAwesomeIcon className="icon" icon={faInstagram} />
+                </a>
+                <a className='footer-icon' href={socialNetworkLinks.facebook} target='_blank'>
+                    <FontAwesomeIcon className="icon" icon={faFacebook} />
+                </a>
```
Comment (VolodymyrTrach) [inline client/src/components/app-footer-links/App-footer-links.js]: fix this warning  "Using target="_blank" without rel="noopener noreferrer" is a security risk"

**Pull request closed without merging**